backend/app/models.py

```diff
@@ -34,6 +35,7 @@ class Hit(BaseModel):
     id: str
     title: str
     poster_path: str
+    imdb_rating: str | None

```

Comment:
Remember to run the TypeScript interface script

Reply:
I added it manually 🤷🏽